<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LRWidget</class>
 <widget class="QWidget" name="LRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LRWidget</string>
  </property>
  <widget class="QTabWidget" name="m_lRTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="loginTab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="phoneLabel">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>30</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>75</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_phoneLineEdit">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>30</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>75</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_loginPushButton">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>170</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_registerPushButton">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>170</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>前往注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="vfCodeLabel">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>120</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>验证码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_vfCodeLineEdit">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>120</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_getVfCodepushButton">
     <property name="geometry">
      <rect>
       <x>257</x>
       <y>120</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="registerTab">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="vfCodeLabel_r">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>155</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>验证码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_passwordLineEdit_r">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>65</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel_r">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>65</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneLabel_r">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>20</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手机号：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_getVfCodepushButton_r">
     <property name="geometry">
      <rect>
       <x>257</x>
       <y>155</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_vfCodeLineEdit_r">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>155</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_phoneLineEdit_r">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>20</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_registerPushButton_r">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>200</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="rePasswordLabel_r">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>110</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>re密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_rePasswordLineEdit_r">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>110</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_loginPushButton_r">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>200</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;
background-color: #ffffff;</string>
     </property>
     <property name="text">
      <string>返回登录</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
